src/models/customs-declarations.js

```diff
@@ -22,6 +22,23 @@ const extractDocumentReferenceId = (documentReference) => {
   return match.length === 1 ? match[0] : null
 }
 
+const determineWeightOrQuantity = (commodity, decisions) => {
+  // C640/CHEDA decisions should show the quantity, otherwise the weight
+  const hasC640Decision = decisions.some(decision => {
```

Comment:
Think we could maybe remove the comment and rename `hasC640Decision` to `showQuantity` and it becomes a bit more self explanatory.